pr-deploy/src/github.ts

```diff
@@ -145,7 +144,6 @@ export class PullRequest {
           'Please click the `Create a test site` button above to ' +
           'deploy the minified build of this PR along with examples from ' +
           '`examples/` and `test/manual/`. It should only take a minute.',
-        text: `Commit SHA: ${sha}`,
```

Comment:
For usability / debuggability, would it makes sense to retain this `text` field in `buildCompleted()`, but not actually scrape it `initializeDeployment()`?

Reply:
the commit sha is displayed as part of the Checks UI on GH